packages/provider/src/core.ts

```diff
@@ -188,6 +189,54 @@ export class Keplr implements IKeplr {
     );
   }
 
+  async signEthereum(
+    chainId: string,
+    signer: string,
+    signDoc: {
+      bodyBytes?: Uint8Array | null;
+      authInfoBytes?: Uint8Array | null;
+      chainId?: string | null;
+      accountNumber?: Long | null;
+    },
+    signOptions: KeplrSignOptions = {}
+  ): Promise<DirectSignResponse> {
+    // How to access mnemonic to re-generate Eth key...
```

Comment:
this part runs in the content script, and there could be some potential security risk to handling sensitive data on the content script. You shouldn't create a msg that returns a mnemonic. This should only be processed in the background.

Reply:
Yeah, the mnemonic is only accessed within the keyring file, it shouldn't be exposed within the content script right now